### ISSUE TITLE
Handle focusing one rows per page and next/prev page

### ISSUE THREAD

**Issue by haxxmaxx** [qlik-oss/sn-table#136]
Currently these buttons are always focusable. If you haven't enter the table yet and tab, you can access them. If we want to follow the same patterns as other charts in sense, they should have tabindex -1 when outside the table, then manually handle their focus,
